rest/taskana-rest-spring/src/main/java/pro/taskana/workbasket/rest/models/WorkbasketSummaryRepresentationModel.java

```diff
@@ -9,21 +9,46 @@
 public class WorkbasketSummaryRepresentationModel
     extends RepresentationModel<WorkbasketSummaryRepresentationModel> {
 
+  /** Unique ID. */
   protected String workbasketId;
+  /** the professional key for the workbasket. */
```

Comment:
professionel key? Is this different to the key fields in other classes?

Reply:
professional = fachlich. I have no idea why I added this here. Shall I remove it?